test/quantization/test_quantize_fx.py

```diff
@@ -757,3 +768,177 @@ def forward(self, x):
             quantized,
             expected_node_occurrence=count_check,
             expected_node_list=order_check)
+
+class TestQuantizeFxModels(QuantizationTestCase):
```

Comment:
Should we have this test in torchvision repo instead? We had this test previously: https://github.com/pytorch/vision/blob/master/test/test_quantized_models.py

Reply:
does torchvision depend on pytorch master?